### PR TITLE
Sets the default FPS to 63

### PR DESCRIPTION
**Pull request by hal9000PR**
## What Does This PR Do
Title

## Why It's Good For The Game
New players can have a smoother experience without having to wait for someone on the discord to tell them about the FPS option.

## Changelog
:cl:
tweak: new players FPS is defaulted to 63 now. 
/:cl:

### CODE REVIEW

**Comment by Contrabang** [comment]
Please. I went 750 hours without knowing this.
Also something about nearest neighbor would also be helpful somewhere, but that might be a wiki thing.

File: SQL/paradise_schema.sql [hunk 1]
```diff
@@ -277,7 +277,7 @@ CREATE TABLE `player` (
   `volume_mixer` longtext COLLATE utf8mb4_unicode_ci DEFAULT NULL,
   `lastchangelog` varchar(32) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '0',
   `exp` longtext COLLATE utf8mb4_unicode_ci DEFAULT NULL,
-  `clientfps` smallint(4) DEFAULT '0',
+  `clientfps` smallint(4) DEFAULT '63',
```
Comment (AffectedArc07) [inline SQL/paradise_schema.sql:280]: You must follow standard SQL update procedures, including:
- Bumping schema number in `code/__DEFINES/misc.dm#370`
- Bumping schema number in the default config at `config/example/config.toml#148`
- Creating an update file in `SQL/updates/34-35.sql` with this change that can be applied to the production schema

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]

File: SQL/updates/34-35.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Updates DB from 34 to 35 -hal9000PR
+# Changes DB default for clientFPS to 63
+
+ALTER TABLE `player` CHANGE COLUMN `clientfps` `clientfps` INT(11) DEFAULT '63' AFTER `exp`;
```
Comment (SteelSlayer) [inline SQL/updates/34-35.sql:4]: The schema is set for a `smallint(4)`, but here you have `int(11)`. Intended?

Also side note, do we need the `AFTER col_name` syntax here? The query appears to works fine without it. I noticed the update queries we have do tend to use this though. This is more something for Affected can answer.

Comment (AffectedArc07) [reply SQL/updates/34-35.sql:4]: `Also side note, do we need the AFTER col_name syntax here? The query appears to works fine without it. I noticed the update queries we have do tend to use this though. This is more something for Affected to answer.`

I prefer having it like this for maximum sanity. And yes it should be `smallint(4)`.

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**